### ISSUE TITLE
Disease cure duration

### ISSUE THREAD

**Issue by Mystikvm** [Mythos/RimWorld-Diseases-Overhauled#9]
For some diseases the cure durations are way overblown. Asthma has a cure duration of 420 hours, which is over two weeks. If the doctor botched the cure, the disease will progress faster and the pawn will die before your next cure attempt. In terms of gameplay these are not fun at all. You need to get at least one more cure to fix the botch, so cure durations need to be rebalanced.

This all happened because I misjudged the ingame duration of days. They are actually a lot shorter than they feel when playing. Also, the game previously measured cure durations in ticks, now it just uses in-game hours. Not knowing exactly how long a tick took in-game I was not able to do an exact conversion.

I will try to devote some time this week to rebalancing them. I have already made some local changes to my game that work a lot better. Goal is that, except for the most serious afflictions, after a cure attempt with a 0% effectivity you should at least get one more attempt.

**Comment by Mythos** [comment]
I agree that some diseases last way longer than before.
We should play a bit around with those values as it shouldn't be too easy, but also not impossible to heal pawns.

Here's a list of the affected changes:

https://github.com/Mythos/RimWorld-Diseases-Overhauled/commit/2021133482a5f7f99eea3bb15f556069f164b3d4#diff-203570eaf1d218196b659cc60d4af05b

https://github.com/Mythos/RimWorld-Diseases-Overhauled/commit/2021133482a5f7f99eea3bb15f556069f164b3d4#diff-a6a701f854ea77493ffc4e8db86c943c

https://github.com/Mythos/RimWorld-Diseases-Overhauled/commit/2021133482a5f7f99eea3bb15f556069f164b3d4#diff-a309387d2c9c4b4d4d442f6ebadd1621

https://github.com/Mythos/RimWorld-Diseases-Overhauled/commit/2021133482a5f7f99eea3bb15f556069f164b3d4#diff-0cf56085851c19237502c1a65a6908d9

https://github.com/Mythos/RimWorld-Diseases-Overhauled/commit/2021133482a5f7f99eea3bb15f556069f164b3d4#diff-adc04942a528945b34cf970ac9828b56

https://github.com/Mythos/RimWorld-Diseases-Overhauled/commit/2021133482a5f7f99eea3bb15f556069f164b3d4#diff-069e80a0331fb86758ca7e4968806b0c

https://github.com/Mythos/RimWorld-Diseases-Overhauled/commit/2021133482a5f7f99eea3bb15f556069f164b3d4#diff-a43aa002ce9d6979ed2d5cbafef86da8

https://github.com/Mythos/RimWorld-Diseases-Overhauled/commit/2021133482a5f7f99eea3bb15f556069f164b3d4#diff-995b7cb12a4cc2193b3ebd871e8cd69e

https://github.com/Mythos/RimWorld-Diseases-Overhauled/commit/2021133482a5f7f99eea3bb15f556069f164b3d4#diff-31be124eceec3fb99aa7fc01b8532ad4

https://github.com/Mythos/RimWorld-Diseases-Overhauled/commit/2021133482a5f7f99eea3bb15f556069f164b3d4#diff-d59af31632c3c513d92b5143a2b9972d

https://github.com/Mythos/RimWorld-Diseases-Overhauled/commit/2021133482a5f7f99eea3bb15f556069f164b3d4#diff-9d68489932dd2b429516d84b009c67b9

https://github.com/Mythos/RimWorld-Diseases-Overhauled/commit/2021133482a5f7f99eea3bb15f556069f164b3d4#diff-3d8a2cf581c86aeee1b55e672356423e

https://github.com/Mythos/RimWorld-Diseases-Overhauled/commit/2021133482a5f7f99eea3bb15f556069f164b3d4#diff-6bc864a1d49a4e2b54898ff7db6be5e4

**Comment by Mystikvm** [comment]
I made some changes locally. I'm not very experienced with git, I have changes lined up in my local repo. Do I make a PR? Or do I push them to the remote develop branch?

In any case: most of the tendDurationHours have been modified to make it a little more fair. In some cases I have also changed the severity increase to be a little slower. I have made changes to the Brain Worm and Heart Worm, making them functionally the same as Gut Worms. That is to say, the disease is cured after a total tend quality of 5. 

In general, most diseased pawns can withstand one low quality cure. A second low quality cure might kill the pawn. This is more fair than before where a single low quality cure could mean a death sentence. This was especially harsh in the early game and should address user complaints on Steam that the mod adds complexity but is a little too unfair in some ways.
